### ISSUE TITLE
[MNG-6669] Tycho cannot resolve project dependencies 

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#8566]
**[Mark McKelvy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mckelvym)** opened **[MNG-6669](https://issues.apache.org/jira/browse/MNG-6669?redirect=false)** and commented

Building a project that uses the Tycho plug-in with Maven version 3.6.1 fails because dependencies cannot be resolved.

Building the same project in Maven version 3.6.0 succeeds.

I've tested using 'mvn clean' as well as 'mvn install' with similar results.

```none
10:31:21 [jem_libs-updates-3] $ mvn -f org.jfree.jfreechart/pom.xml install -P packaging-eclipse
10:31:22 [INFO] Scanning for projects...
10:31:24 [INFO] Computing target platform for MavenProject: org.jfree.jfreechart:org.jfree.jfreechart:1.5.0 @ /home/tomcat/jenkins/workspace/jem_libs-updates-3/org.jfree.jfreechart/pom.xml
10:31:24 [INFO] Resolving dependencies of MavenProject: org.jfree.jfreechart:org.jfree.jfreechart:1.5.0 @ /home/tomcat/jenkins/workspace/jem_libs-updates-3/org.jfree.jfreechart/pom.xml
10:31:24 [INFO] {osgi.os=win32, osgi.ws=win32, org.eclipse.update.install.features=true, osgi.arch=x86_64}
10:31:24 [ERROR] Cannot resolve project dependencies:
10:31:24 [ERROR]   Software being installed: org.jfree.jfreechart 1.5.0
10:31:24 [ERROR]   Missing requirement: org.jfree.jfreechart 1.5.0 requires 'bundle org.eclipse.swt 0.0.0' but it could not be found
10:31:24 [ERROR] 
10:31:24 [ERROR] See http://wiki.eclipse.org/Tycho/Dependency_Resolution_Troubleshooting for help.
10:31:24 [ERROR] Cannot resolve dependencies of MavenProject: org.jfree.jfreechart:org.jfree.jfreechart:1.5.0 @ /home/tomcat/jenkins/workspace/jem_libs-updates-3/org.jfree.jfreechart/pom.xml: See log for details -> [Help 1]
10:31:24 [ERROR] 
10:31:24 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
10:31:24 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
10:31:24 [ERROR] 
10:31:24 [ERROR] For more information about the errors and possible solutions, please read the following articles:
10:31:24 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
```

Please see the attached project.

---

**Affects:** 3.6.1

**Attachments:**
- [org.jfree.jfreechart.zip](https://issues.apache.org/jira/secure/attachment/12970714/org.jfree.jfreechart.zip) (_3.50 kB_)

**Issue Links:**
- [MNG-6642](https://issues.apache.org/jira/browse/MNG-6642) Tycho-based modules do not build with 3.6.1 (works with 3.6.0)
 (_**"duplicates"**_)

1 votes, 3 watchers

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

This seems to work with 3.6.2-SNAPSHOT for me. Fails with 3.6.1. Can you confirm?

**Comment by jira-importer** [comment]
**[Mark McKelvy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mckelvym)** commented

Apologies - where can I download 3.6.2-SNAPSHOT?

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Take this one from Jenkins: https://repository.apache.org/content/repositories/snapshots/org/apache/maven/apache-maven/3.6.2-SNAPSHOT/apache-maven-3.6.2-20190725.095335-35-bin.tar.gz

**Comment by jira-importer** [comment]
**[Mark McKelvy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=mckelvym)** commented

Thanks, Michael.

I can confirm it works with 3.6.2-SNAPSHOT.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Already fixed with 3.6.2.
